doc/releases/changelog-dev.md

```diff
@@ -23,7 +23,10 @@
   >>> qml.CNOT((0,1))
   CNOT(wires=[0, 1])
   ```
+  
+* Instead of checking types, objects are processed in `QuantumTape`'s based on a new `_queue_category` property.
```

Comment:
```suggestion
* Instead of checking types, objects are processed in `QuantumTape`'s based on a new `_queue_category` property.
  [(#2408)](https://github.com/PennyLaneAI/pennylane/pull/2408)
```